### ISSUE TITLE
Something about your update part

### ISSUE THREAD

**Issue by ghost** [OlavHN/bnlstm#9]
Thanks for your implementation.

When I try your code, I first calculate the validation error after each iteration. Then I only calculate the error after all iterations. And the accuracy is far low in the second case.

The iteration is set to be 1000 in both cases. And I use the same validation data in each iteration.

Does it mean the code still updates the moving moments in testing(i.e. feeding train=False)?

**Comment by LiangSun1994** [comment]
@ghost I have reimplemented this bn-lstm code in TensorFlow. However, I got this similar result. I think we can discuss in here. I have pasted the picture of accuracy in the train and test data set.
[Discussion](https://github.com/cooijmanstim/recurrent-batch-normalization/issues/7)
